### ISSUE TITLE
I cannot open Request ..... but thats what I mean

### ISSUE THREAD

**Issue by cadefoster117** [brichard19/BitCrack#156]
Add a signature bruteforce.... for a given S ... and given r and z and brute s and s + 1  ... to fit the signature.... 
if you wish can give you python 5 line script that I try to make it work with cuda :)
ask and will be given ;)

**Comment by SatoshiNakamotoBitcoins** [comment]
@cadefoster117 That would be a nice upgrade...

Something like this...https://github.com/qubd/mini_ecdsa

**Comment by cadefoster117** [comment]
Yep... specify z r and public key... give value of s just like --keyspace and count + 1 to see if signature is valid if not count +1 again ;)

**Comment by SatoshiNakamotoBitcoins** [comment]
@cadefoster117 sounds like magic...looking forward to your repository...
